tests/test.py

```diff
@@ -4529,310 +4242,61 @@ class TestZoneIDDecryptionWithVaultMasterKeyLoader(HashiCorpVaultMasterKeyLoader
+                decrypted = deserialize_and_decrypt_acrastruct(response, client_id_private_key, client_id)
                 self.assertEqual(data, decrypted)
-            # wait decryption error with incorrect client id
-            with ProcessContextManager(self.fork_connector_for_translator(connector_port2, translator_port, incorrect_client_id)):
-                response = request_func(connector_port2, incorrect_client_id, None, None)
-                self.assertNotEqual(data, response)
-
-    def apiDecryptionTest(self, request_func, use_http=False, use_grpc=False):
-        # one is set
-        self.assertTrue(use_http or use_grpc)
-        # two is not acceptable
-        self.assertFalse(use_http and use_grpc)
-        translator_port = 3456
-        connector_port = 12345
-        connector_port2 = connector_port+1
-        client_id = "keypair1"
-        data = get_pregenerated_random_data().encode('ascii')
-        encryption_key = read_storage_public_key(
-            client_id, keys_dir=KEYS_FOLDER.name)
-        acrastruct = create_acrastruct(data, encryption_key)
-
-        zone = zones[0]
-        incorrect_zone = zones[1]
-        zone_public = b64decode(zone['public_key'].encode('ascii'))
-        acrastruct_with_zone = create_acrastruct(
-            data, zone_public, context=zone['id'].encode('ascii'))
-        connection_string = 'tcp://127.0.0.1:{}'.format(translator_port)
-        translator_kwargs = {
-            'incoming_connection_http_string': connection_string if use_http else '',
-            # turn off grpc to avoid check connection to it without acra-connector
-            'incoming_connection_grpc_string': connection_string if use_grpc else '',}
-
-        correct_client_id = 'keypair1'
-        incorrect_client_id = 'keypair2'
-        with ProcessContextManager(self.fork_translator(translator_kwargs)):
-            with ProcessContextManager(self.fork_connector_for_translator(connector_port, translator_port, client_id)):
-                response = request_func(connector_port, correct_client_id, None, acrastruct)
-                self.assertEqual(data, response)
-
-                # test with correct zone id
-                response = request_func(
-                    connector_port, client_id, zone['id'], acrastruct_with_zone)
-                self.assertEqual(data, response)
-
-                # test with incorrect zone id
-                response = request_func(
-                    connector_port, client_id, incorrect_zone['id'],
-                    acrastruct_with_zone)
-                self.assertNotEqual(data, response)
-
-            # wait decryption error with incorrect client id
-            with ProcessContextManager(self.fork_connector_for_translator(connector_port2, translator_port, incorrect_client_id)):
-                response = request_func(connector_port2, incorrect_client_id, None, acrastruct)
-                self.assertNotEqual(data, response)
-
-    def testHTTPApiResponses(self):
```

Comment:
here too. we can test it with `https://` and without connector

Reply:
Will return this test